rwengine/src/loaders/LoaderCutsceneDAT.cpp

```diff
@@ -16,17 +18,21 @@ void LoaderCutsceneDAT::load(CutsceneTracks &tracks, const FileHandle& file) {
     std::string dataStr(file->data, file->length);
     std::stringstream ss(dataStr);
 
+    std::vector<std::string> lineParts;
+
     int numZooms = 0;
     ss >> numZooms;
     ss.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
 
     for (int i = 0; i < numZooms; ++i) {
-        float t = 0.f;
-        float z = 0.f;
-        ss >> t;
-        ss.ignore(2, ',');
-        ss >> z;
-        ss.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
+        std::string line;
+        ss >> line;
+
+        boost::split(lineParts, line, boost::is_any_of(","));
```

Comment:
Wasn't there once some endeavor (#317) for getting rid of the boost dependency, at least for anything except the testcases? 

(IMHO, I do not really see the advantage here over using e.g. `std::getline` with `,` as separator, or just fixing the existing code.)